### DISCUSSION TITLE
有点不是很明白 xreactions的 fulfill 作用是什么，并且，是不是打错字了。应该是 fullfill吗

### DISCUSSION THREAD

**Discussion by ZhihaoLau** [alibaba/formily#2543] [Q&A 2.x] [answered]
如题

**Comment by janryWang** [comment, marked as answer]
https://react.formilyjs.org/zh-CN/api/shared/schema

**Reply by quanru** [reply]
这个文档的官网入口在哪？文档有点深了

**Reply by liucy1983** [reply]
访问https://v2.formilyjs.org/, 上边导航栏Basic Core Library，选react或者vue，会跳到https://react.formilyjs.org/或者https://vue.formilyjs.org/。
上边导航栏点API，右侧导航栏点Shared，然后选Schema就是了
